JobConfig/primary/FlateMinus.fcl

```diff
@@ -8,7 +8,7 @@ physics.producers.generate : {
   module_type : FlatMuonDaughterGenerator
   inputSimParticles: TargetStopResampler
   stoppingTargetMaterial : "Al"
-  pdgId : 13
+  pdgId : 11
```

Comment:
is this (and the flatePlus.fcl change) a bug fix?   Or does it go together with a change in the generator code?